File: 1-fundamentos-e-logica/9-exercicios-de-logica/Desafios/logica-pensamento-computacional/calculo_salario.cs
using System;

public class Desafio
{
    public static void Main()
    {
        // Lê os valores de Entrada:
        float valorSalario = float.Parse(Console.ReadLine());
        float valorBeneficios = float.Parse(Console.ReadLine());

        float valorImposto = 0;

        if (valorSalario >= 0 && valorSalario <= 1100)
        {
            // Atribui a alíquota de 5% mediante o salário:
            valorImposto = 0.05F * valorSalario;
        }
        else if (valorSalario <= 2500)
        {
            // Alíquota de 10% para salários entre 1100.01 e 2500
            valorImposto = 0.10F * valorSalario;
        }
        else
        {
            // Alíquota de 15% para salários acima de 2500
            valorImposto = 0.15F * valorSalario;
        }

        // Calcula e imprime a Saída (com 2 casas decimais):
        float saida = valorSalario - valorImposto + valorBeneficios;
        Console.WriteLine(saida.ToString("0.00"));
    }
}

File: 1-fundamentos-e-logica/9-exercicios-de-logica/Desafios/logica-pensamento-computacional/CalculoSalario.java
// - new Scanner(System.in): cria um leitor de Entradas, com métodos úteis com prefixo "next";
// - System.out.println: imprime um texto de Saída (Output) e pulando uma linha

import java.util.Scanner;

public class Desafio {
    
    public static void main(String[] args){
        // Lê os valores de Entrada:
        Scanner leitorDeEntradas = new Scanner(System.in);
        float valorSalario = leitorDeEntradas.nextFloat();
        float valorBeneficios = leitorDeEntradas.nextFloat();

        float valorImposto = 0;
        
        if (valorSalario >= 0 && valorSalario <= 1100) {
            // Atribui a aliquota de 5% mediante o salário:
            valorImposto = 0.05F * valorSalario;
        } else if (valorSalario <= 2500) {
            // Atribui a aliquota de 10% mediante o salário:
            valorImposto = 0.10F * valorSalario;
        } else {
            // Atribui a aliquota de 15% mediante o salário:
            valorImposto = 0.15F * valorSalario;
        }

        // Calcula e imprime a Saída (com 2 casas decimais):
        float saida = valorSalario - valorImposto + valorBeneficios;
        System.out.println(String.format("%.2f", saida));
    }
}
